godot-core/src/builtin/string/mod.rs

```diff
@@ -54,6 +54,20 @@ impl FromGodot for String {
     }
 }
 
+// ----------------------------------------------------------------------------------------------------------------------------------------------
+// Encoding
+
+/// Specifies string encoding.
+///
+/// Used in functions such as [`GString::try_from_bytes()`][GString::try_from_bytes] to handle multiple input string encodings.
+#[non_exhaustive]
+#[derive(Copy, Clone, Eq, PartialEq, Debug)]
```

Comment:
maybe include `Default`? `utf8` would make sense to me as it's the default in rust elsewhere.

Reply:
This would lead to code such as:
```rs
GString::try_from_bytes(bytes, Default::default())
```
which I find much less readable than:
```rs
GString::try_from_bytes(bytes, Encoding::Utf8)
```
<br>
Since UTF-8 isn't a 100% obvious default (Godot itself uses a mix of Latin-1 and UTF-32), I'd rather make this explicit.